homeassistant/components/ohme/config_flow.py

```diff
@@ -0,0 +1,59 @@
+"""Config flow for ohme integration."""
+
+from typing import Any
+
+from ohme import OhmeApiClient
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.helpers.selector import (
+    TextSelector,
+    TextSelectorConfig,
+    TextSelectorType,
+)
+
+from .const import DOMAIN
+
+USER_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_EMAIL): TextSelector(
+            TextSelectorConfig(
+                type=TextSelectorType.EMAIL,
+                autocomplete="email",
+            ),
+        ),
+        vol.Required(CONF_PASSWORD): TextSelector(
+            TextSelectorConfig(
+                type=TextSelectorType.PASSWORD,
+                autocomplete="current-password",
+            ),
+        ),
+    }
+)
+
+
+class OhmeConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Config flow."""
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """First config step."""
+
+        errors: dict[str, str] = {}
+
+        if user_input is not None:
+            self._async_abort_entries_match({CONF_EMAIL: user_input[CONF_EMAIL]})
+
+            instance = OhmeApiClient(user_input[CONF_EMAIL], user_input[CONF_PASSWORD])
+            if not await instance.async_login():
+                errors["base"] = "auth_error"
```

Comment:
```suggestion
                errors["base"] = "invalid_auth"
```
is what it's usually valled and there's also a default string available